### ISSUE TITLE
inventory editing?

### ISSUE THREAD

**Issue by shadow4378a** [Offroaders123/Dovetail#2]
i dont really use github so i dont know where to actually send this, but i was wondering if you were planning on adding inventory adding or not?

**Comment by Offroaders123** [comment]
Yeah eventually I want to get inventory/full world NBT editing setup (editing chunks and other things like that). First I just have to get world format parsing setup in the browser, like being able to load Java world `.zip`/`.mca` files, and Bedrock `.mcworld` files, which also means parsing the world LevelDB database, which is where all of the world data is stored.

And thanks for opening this, it was the perfect spot to keep track of features and such.

**Comment by ghost** [comment]
Hey, didnt know where to put this, but I wanted to say thanks. Using the website I was able to restore my correct inventory in a world and i didnt know what id do without it. I especially appreciate the ability to edit it by text. Thank you for making it.
